packages/auth/migrations/20220504161932_create_grants_table.js

```diff
@@ -15,10 +15,6 @@ exports.up = function (knex) {
     table.string('clientNonce')
     table.string('client').notNullable()
 
-    table.string('interactId').unique()
-    table.string('interactRef').unique()
-    table.string('interactNonce').unique()
-
```

Comment:
Since testnet has deployed we've moved away from editing the migration files directly and instead making a new migration to update the tables. https://github.com/interledger/rafiki/pull/1522#discussion_r1245523191

I guess in this case it's technically not going to break anything because we are just removing fields here. Testnet would just still have them. Still, I think we should put them in a new migration IMO.